### DISCUSSION TITLE
API Pagination - Starting Page Number

### DISCUSSION THREAD

**Discussion by dkaiping** [Budibase/budibase#7845] [Help] [answered]
Hi all - I've been exploring Budibase for just under a week now as a way to display and manage some simple screens for our business.  I have a series of Pageable Rest APIs, and I have been trying to leverage the "Paginate" functionality in the table, against an API that is also pagingation enabled.

The issue I'm having - is all of my pageable endpoints the page size starts at 0 but the out of the box paging starts the page at page=1.  Is there a way to tell the paging functionality to do page-1?  I really like the simplicity of the "enable paging" and way hoping not to have to manually create buttons to handle paging. Thanks in advance for any thoughts.

![image](https://user-images.githubusercontent.com/7794557/191093182-a78dad25-1c48-431a-9fbb-f93221179848.png)

**Comment by melohagan** [comment, marked as answer]
Feature request her: https://github.com/Budibase/budibase/issues/7978

Need to add bindings to the pagination to allow custom numbering.
